src/components/util/document-store.tsx

```diff
@@ -0,0 +1,36 @@
+import { connectWallet, getWalletNetwork } from "./wallet";
+
+export const getDocumentStores = async (): Promise<Array<string>> => {
+  try {
+    const provider = await connectWallet();
+    const logInfo = {
+      fromBlock: "earliest",
+      toBlock: "latest",
+      topics: [
+        "0x8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e0",
```

Comment:
what's this ? might be worth to add a comment on what it is

Reply:
done.

Reply:
does that mean we need to update every time there is a new version of a document store?